org.codeberg.dnkl.foot.yml

```diff
@@ -0,0 +1,60 @@
+app-id: org.codeberg.dnkl.foot
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: foot.sh
+
+finish-args:
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - name: libutf8proc
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://github.com/JuliaStrings/utf8proc/archive/refs/tags/v2.7.0.zip
+        sha256: 03f6a8e5d22c05d719d44d4fa5692bd8b53624da77905462e3405ada19fa5719
+  - name: tllist
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://codeberg.org/dnkl/tllist/archive/1.1.0.zip
+        sha256: 9eb339e668d50e44ac1ffef825e8d17948e199d05840fd0e77775324ce3ffb4d
+  - name: fcft
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://codeberg.org/dnkl/fcft/archive/3.1.2.zip
+        sha256: 9b599541c35dedde4ce32f84b8ec226ba33b10a3c188440ad55f315a401ef2b8
+  - name: foot
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://codeberg.org/dnkl/foot/archive/1.13.0.zip
+        sha256: d0af98e61ea25e121549ded561ee1fe1b11c901b81a7c11844bd6fb32139f256
+  - name: foot.sh
+    buildsystem: simple
+    sources:
+      - type: file
+        path: ./foot.sh
+    build-commands:
+      - install -Dm755 ./foot.sh -t /app/bin/
+  - name: host-spawn
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+      build-args:
+        - --share=network
```

Comment:
build-args are NOT allowed - it should have failed if you tried locally. You need to list the dependencies for go. See https://github.com/flatpak/flatpak-builder-tools for helpers.

Reply:
Ok I removed the network build arg. Now it builds `host-spawn` from source and without network. I hope now it will be ok.

And the appstream file is on it's way https://codeberg.org/dnkl/foot/pulls/1149